src/components/Banner/examples/basic.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import Banner from '../Banner';
+import classNames from 'classnames';
+
+const {
+	PropTypes: {
+		number,
+		string
+	}
+} = React;
+
+
+const InfoIcon = React.createClass({
+	propTypes: {
+		className: string,
+		color: string,
+		size: number,
```

Comment:
maybe we need to name this size prop something different since other components use `size: oneOf(['small', 'normal', 'large']) while this one expects a number. i think we should be more consistent.